client/app/mobile/modules/series/root/MasterSeries.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { View, Text } from 'react-native'
+
+export const MasterSeries = () => 
```

Comment:
Try not mixing named exports and default exports. Be consistent. For example, for presentational named and hocs as default or as your own.

Reply:
Mobile side has not been develope/modify yet